src/main/main.scss

```diff
@@ -0,0 +1,53 @@
+@import "./variables.scss";
```

Comment:
Remove this catalog `src/main` with all files. And move this file to the `src/index.scss`.

Reply:
Remove `@import "./variables.scss"` from this file. Webpack already imports it for you.